conjurapi/variable_test.go

```diff
@@ -120,7 +121,7 @@ func TestClient_RetrieveSecret(t *testing.T) {
 					variableIds = append(variableIds, id)
 				}
 
-				secrets, err := conjur.RetrieveBatchSecrets(variableIds)
+				secrets, err := conjur.RetrieveBatchSecrets(variableIds, true)
```

Comment:
can we add some additional test cases here?
- retrieving batch secrets with no base64 arg
- retrieving batch secrets with base64 arg false

is this test also running against the v4 Conjur? wondering how it passes in that case, since you're sending a non-nil second param